docs/docs/logging.md

```
@@ -0,0 +1,246 @@
+Authorization events tracking access to catalog resources. **Contains PII** (user identities).
+
+**Identified by:** `"event_source": "audit"`
+
+**Structure:**
+
+| Field                  | Type            | Description                       |
+|------------------------|-----------------|-----------------------------------|
+| `event_source`         | String          | Always `"audit"`                  |
+| `action` or `actions`  | String or Array | Operation(s) attempted (e.g., `"read_data"`, `"drop"`, `"CreateNamespace"`) |
+| `entity` or `entities` | Object or Array | Resource(s) accessed, containing `entity_type` and type-specific fields (e.g., `warehouse-id`, `namespace`, `table`) |
+| `actor`                | Object          | Who performed the action (see [Actor Types](#actor-types)) |
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)